plugins/ConditionalRestrictions.py

```diff
@@ -24,14 +24,24 @@
 import re
 from datetime import date
 from modules.Stablehash import stablehash64
+from plugins.TagFix_Opening_Hours import TagFix_Opening_Hours
 
 class ConditionalRestrictions(Plugin):
   def init(self, logger):
     Plugin.init(self, logger)
 
-    self.ReYear = re.compile(r'20\d\d') # Update in 2099
+    self.ReYear = re.compile(r'\b20\d\d\b') # Update in 2099
     self.ReSimpleCondition = re.compile(r'^\w+$', re.ASCII)
+    self.ReAND = re.compile(r'\band\b', re.IGNORECASE)
     self.currentYear = date.today().year
+    self.comparisonOperatorChars = ["<", "=", ">"]
+    # Following regex are to detect likely (possibly misspelled) opening_hours syntaxes
+    self.ReWeekdayMonthOpeningH = re.compile(r'\b[A-Z][a-z]') # i.e. Mar or Mo
```

Comment:
Comment is about 2 or 3 chars, while only 2.

Reply:
Added a `+`
(For the regex (which isn't used to obtain capture groups, only to detect the presence of such a string) it doesn't change anything, in `Ma` `Mar` and `March` [2 of which are invalid and should be detected] it simply matched the first two chars as it's open-ended)